sdk/liftrqumulo/Azure.ResourceManager.LiftrQumulo/api/Azure.ResourceManager.LiftrQumulo.netstandard2.0.cs

```diff
@@ -0,0 +1,480 @@
+    {
+        protected StorageMoverCollection() { }
+        public virtual Azure.ResourceManager.ArmOperation<Azure.ResourceManager.LiftrQumulo.StorageMoverResource> CreateOrUpdate(Azure.WaitUntil waitUntil, string storageMoverName, Azure.ResourceManager.LiftrQumulo.StorageMoverData data, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.ResourceManager.ArmOperation<Azure.ResourceManager.LiftrQumulo.StorageMoverResource>> CreateOrUpdateAsync(Azure.WaitUntil waitUntil, string storageMoverName, Azure.ResourceManager.LiftrQumulo.StorageMoverData data, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<bool> Exists(string storageMoverName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ExistsAsync(string storageMoverName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.LiftrQumulo.StorageMoverResource> Get(string storageMoverName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.LiftrQumulo.StorageMoverResource> GetAll(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.LiftrQumulo.StorageMoverResource> GetAllAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.LiftrQumulo.StorageMoverResource>> GetAsync(string storageMoverName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        System.Collections.Generic.IAsyncEnumerator<Azure.ResourceManager.LiftrQumulo.StorageMoverResource> System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.LiftrQumulo.StorageMoverResource>.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken) { throw null; }
+        System.Collections.Generic.IEnumerator<Azure.ResourceManager.LiftrQumulo.StorageMoverResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.LiftrQumulo.StorageMoverResource>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class StorageMoverData : Azure.ResourceManager.Models.TrackedResourceData
```

Comment:
The `StorageMover` is another service, to avoid naming collision please prepend the RP name as prefix, for rename model name you can refer this https://github.com/Azure/autorest.csharp/blob/feature/v3/docs/mgmt/polishing.md